src/components/DetailPage/DetailPage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
I love that there is a little dark theme going on so I don't have to start from scratch ❤️ 
![Alt Text](https://media.giphy.com/media/vFKqnCdLPNOKc/giphy.gif)